Index: src/capproc.cc
===================================================================
RCS file: /usr/local/cvsroot/vcr/src/capproc.cc,v
retrieving revision 1.9
diff -u -r1.9 capproc.cc
--- src/capproc.cc	2001/04/02 21:05:19	1.9
+++ src/capproc.cc	2001/06/01 14:31:06
@@ -1,6 +1,7 @@
 #include "capproc.h"
 #include "dsp.h"
 #include <unistd.h>
+#include <stdio.h>
 #include <avifile.h>
 #include <videoencoder.h>
 
